plugins/modules/discovery_rule.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Matthias M Dellweg <dellweg@atix.de> (ATIX AG)
```

Comment:
I think we can safely assume Matthias has not really any more copyright on the result ;)

Reply:
Hehe, I'm not always sure how to do those :-) I did steal his ```location.py``` to start working in this :grimacing: